release/create_release.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -euo pipefail
+
+get_latest_tag(){
+  local repo="$1"
+  local latest=$(curl --silent "https://api.github.com/repos/semaphoreci/$repo/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")')
+  echo $latest | cut -c 2-
+}
+
+#LAV LATEST_ARTIFACT_VERSION
+LAV="$(get_latest_tag 'artifact')"
```

Comment:
Don't use the latest, set a version on the top of the file to a specific version.